tests/framework/integration.go

```diff
@@ -196,3 +196,25 @@ func (c integrationClient) Health() error {
 	}
 	return nil
 }
+
+func (c integrationClient) UserAdd(name, password string, opts config.UserAddOptions) (*clientv3.AuthUserAddResponse, error) {
+	ctx := context.Background()
+	return c.Client.Auth.UserAddWithOptions(ctx, name, password, &clientv3.UserAddOptions{
```

Comment:
Suggest to change `c.Client.Auth.UserAddWithOptions` to `c.Client.UserAddWithOptions` for better readability.

Reply:
I don't have a strong opinion here. Having a 2 layered interface is nice for clarity, grouping and testing. We might want to also introduce it in common framework client.